getting-started-guides/on-prem-cluster/standalone-python.md

```diff
@@ -1,136 +1,115 @@
 Get Started with XGBoost4J-Spark on an Apache Spark Standalone Cluster
 ======================================================================
-This is a getting started guide to XGBoost4J-Spark on an Apache Spark3.0 Standalone Cluster. At the end of this guide, the reader will be able to run a sample Apache Spark Python application that runs on NVIDIA GPUs.
+This is a getting started guide to XGBoost4J-Spark on an Apache Spark 3.0+ Standalone Cluster. At the end of this guide, the user can be able to run a sample Apache Spark Python application that runs on NVIDIA GPUs.
```

Comment:
nit: "can be able" is a little bit strange. Either "be able" or "can" is enough.